File: dayTwo/python/level three/taskFive.py
number = int(input("Enter a number to get its factorial: "))

print(f"\nThe factorial of {number}")
factorial = 1
for count in range(number, 0, -1):
    factorial *= count
    print(f"{count} ", end= " x ")
#    print (factorial)

print("=",factorial)

File: dayTwo/javascript/level three/taskFive.js
const prompt = require("prompt-sync")();

let number = Number(prompt("Enter a number to see it's multiplication table\nEnter number:"))

console.log(`----------------------The factorial of ${number}----------------------\n`)
let factorial = 1;
let count = number
 for(;count >= 1;count--){
    factorial *= count
    console.log(`${count} x`)
}
console.log("=", factorial)
